collectors/cgroups.plugin/sys_fs_cgroup.c

```diff
@@ -1300,6 +1300,11 @@ static inline char *cgroup_chart_id_strdupz(const char *s) {
     char *r = strdupz(s);
     netdata_fix_chart_id(r);
 
+    for (char *d = r; *d; d++) {
+        if (*d == '.')
+            *d = '-';
+    }
```

Comment:
Perhaps a comment with an explanation of why we do it?

Reply:
A citation for the issue probably is enough.

I tested the PR and it is working:

```
"cgroup_fourth-fifth-sixth.cpu_limit": 		{
			"id": "cgroup_fourth-fifth-sixth.cpu_limit",
			"name": "cgroup_fourth_fifth_sixth.cpu_limit",
			"type": "cgroup_fourth-fifth-sixth",
			"family": "cpu",
			"context": "cgroup.cpu_limit",
			"title": "CPU Usage within the limits (cgroup_fourth_fifth_sixth.cpu_limit)",
			"priority": 39999,
			"plugin": "cgroups.plugin",
			"module": "/sys/fs/cgroup",
			"enabled": true,
			"units": "percentage",
			"data_url": "/api/v1/data?chart=cgroup_fourth_fifth_sixth.cpu_limit",
			"chart_type": "line",
			"duration": 270,
			"first_entry": 1619541716,
			"last_entry": 1619541985,
			"update_every": 1,
			"dimensions": {
				"used": { "name": "used" }
			},
			"chart_variables": {
			},
			"green": null,
			"red": null,
			"alarms": {
				"cgroup_10min_cpu_usage": {
					"id": 1619540804,
					"status": "UNINITIALIZED",
					"units": "%",
					"update_every": 60
				}
			},
			"chart_labels": {

			}

		},
```

I am waiting for this problem to be addressed for I approve it.